src/getDescription.js

```diff
@@ -0,0 +1,31 @@
+export function getDescription() {
+  const metaDescription = document.querySelector(`meta[name="description"]`);
+  const metaOgDescription = document.querySelector(
+    `meta[property="og:description"]`
+  );
+
+  let metaDescriptionText = metaDescription
+    .getAttribute("content")
+    .replaceAll(/png|jpg|jpeg|gif/gi, " ");
+  let metaOgDescriptionText = metaOgDescription
+    .getAttribute("content")
+    .replaceAll(/png|jpg|jpeg|gif/gi, " ");
+
+  const metaFilterText = metaDescriptionText
```

Comment:
변수명 중점적으로 보는 것을 원하신다고 하셔서, 저라면 filter 된 것이니까 중간에 형용사를 넣어볼 것도 같아요
근데 이건 진짜 중요한 것은 아닌 것 같기도 해서 마음대로 하셔도 좋을 것 같습니다:)
```suggestion
  const metaFilteredText = metaDescriptionText
```